cves/CVE-2019-9721.yml

```diff
@@ -236,8 +244,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: The file was located in the avcodec folder.
```

Comment:
Same thing with the other CVE, i'm pretty sure saying its in the same file path is not a valid reason. Not doubt that it isn't in the avcodec subsystem but yea